app/helpers/interfaces_helper.rb

```diff
@@ -4,4 +4,8 @@ def interfaces_types
       [_(nic_type.humanized_name), nic_type]
     end
   end
+
+  def virtual_host?(nic)
```

Comment:
Virtual might be misleading since some people also use kickstarts ("bare metal") provisioning on virtualized machines. Would `compute_resource?` be better?